nixos/tests/installer.nix

```diff
@@ -123,6 +123,9 @@ let
       $machine->succeed("type -tP ps | tee /dev/stderr") =~ /.nix-profile/
           or die "nix-env failed";
 
+      # This should do nothing
+      $machine->succeed("nixos-rebuild switch >&2");
```

Comment:
I ended up taking this out since it doesn't really do much, and I don't want to test basic functionality of `nixos-rebuild switch` in this test.